lib/dataContract/stateTransition/DataContractCreateTransition.js

```diff
@@ -99,7 +99,7 @@ class DataContractCreateTransition extends AbstractStateTransitionIdentitySigned
  * @property {number} type
  * @property {RawDataContract} dataContract
  * @property {number|null} signaturePublicKeyId
- * @property {string|null} signature
+ * @property {buffer|string|null} signature
```

Comment:
It supposed to be buffer so I would keep only Buffer. How it can be null?

Reply:
it is null, if it's not signed

Reply:
Make sense. Ok, let's keep just Buffer and null